ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -494,6 +523,7 @@ objects:
         # begin bootstrapping
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
           RHCOS_AMI=ami-0df3f99538fbef10f  # FIXME: assumes AWS_REGION is us-east-1
+          export AWS_BASE_DOMAIN="${BASE_DOMAIN:-origin-ci-int-aws.dev.rhcloud.com}"
```

Comment:
I don't think we need to `export` this, since the only consumers are in this shell script, not its sub-processes.

And I don't think we need an AWS-specific variable; if we want dynamic defaults, we can just assign a default value with:

```sh
echo "Base domain: "${BASE_DOMAIN:=origin-ci-int-aws.dev.rhcloud.com}"
```

or some such.

Reply:
Ah, I see `:=` came up [earlier][1].  Did this not work out for some reason?

[1]: https://github.com/openshift/release/pull/5134#discussion_r326827433

Reply:
Removed export as requested and switched to ${base_domain}.